### ISSUE TITLE
The training condition of the best model

### ISSUE THREAD

**Issue by leilegelei1** [wvangansbeke/Sparse-Depth-Completion#4]
Hi,
It's very nice of you to share your code.

However, I can not reproduce the best model whose RMSE is 802 in the selected val set.

Can you tell me the detail training hyper-parameters of the best model （which loss funtion do you choose? the lr policy? The initialization learning rete ）? And how many epoch it takes to get that model. 

Many thanks!!

**Comment by wvangansbeke** [comment]
Hi @leilegelei1,

I use a learning rate of 1e-3 with a batch size of 7. I also use the plateau scheduler of Pytorch during training. I trained the model for 60 epochs. The default values of the hyperparameters are normally correct  and the loss function too. I can train it another time if I have time to double check. However, if I use de weights of my trained model (which you can download), I get a RMSE of 802mm. So the model seems fine. Maybe try to load the pretrained model and see if you get the same results or check the dataloading process.

Kind regards,
Wouter

**Comment by leilegelei1** [comment]
@wvangansbeke 
Thanks!

**Comment by wvangansbeke** [comment]
Hi @leilegelei1,

I've heard from two persons that they obtained my reported numbers exactly with this code. So it should be fine.

Best,
Wouter
